src/vs/editor/browser/services/bulkEditService.ts

```diff
@@ -76,6 +76,7 @@ export interface IBulkEditOptions {
 	undoRedoGroupId?: number;
 	confirmBeforeUndo?: boolean;
 	respectAutoSaveConfig?: boolean;
+	minResourcesToAutosave?: number;
```

Comment:
I believe that I didn't understand this yet. The idea of the bulk edit service is to auto-save (iff enabled) everything unless what's currently opened in the editor anyways (since normal auto-save will pick it up)